generators/client/index.js

```diff
@@ -209,7 +209,7 @@ module.exports = class extends BaseBlueprintGenerator {
 
                 // Application name modified, using each technology's conventions
                 this.camelizedBaseName = _.camelCase(this.baseName);
-                this.angularAppName = this.getAngularAppName();
+                this.frontendAppName = this.getFrontendAppName();
```

Comment:
I believe this will break the downstream 
- core modules
- blueprint modules
- Jhipster modules

Should I retain the assignment here and only focus the removal of usages of `angularAppName` in this PR?

Reply:
I think you can remove, jhipster 7 is to do our cleanup.